crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -1006,6 +1011,10 @@ impl<
     }
 }
 
+/// Type alias for the complex future type used in `RangeBlockMode`
```

Comment:
we could mention here that this is used to manage parallel receipt fetching tasks